### ISSUE TITLE
STATIC_URL variable not being passed?

### ISSUE THREAD

**Issue by kkaczmarczyk** [tiangolo/uwsgi-nginx-docker#54]
I am extending python3.5 version of this image and I am trying to make STATIC_URL point to the directory with statics that Django creates. I seem to not able to do so with python3.5 version. Is there support yet for this option? Should I just overwrite whole nginx configuration?

**Comment by tiangolo** [comment]
Yep, this image doesn't support `STATIC_URL` at least yet, I'll take it as a feature request :slightly_smiling_face:

**Comment by kkaczmarczyk** [comment]
Yeah, I figured so - the VS Code guide was a bit misleading. I used custom nginx configuration and it worked well. However, as you say this feature would be a nice-to-have :)

FWIW I think this guy https://github.com/tiangolo/uwsgi-nginx-flask-docker has it.

**Comment by tiangolo** [comment]
Hehe, yep, that's also me. This image was originally made to work as the base for that one, but also be generalizable to other frameworks apart form Flask (like Django).

**Comment by gozzarda** [comment]
Hi @tiangolo,

Is there a technical reason this functionality can't be included in this more general image? I have found your flask-specific image very useful, and am now trying to deploy a Djano application with static files, and would really appreciate this functionality.

**Comment by tiangolo** [comment]
Hey there @gozzarda, I'm currently not using this image myself, so I wouldn't add new features to it.

Also, you probably don't need this image: https://github.com/tiangolo/uwsgi-nginx-docker#-warning-you-probably-dont-need-this-docker-image

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
